pennylane/qchem/dipole.py

```diff
@@ -205,7 +206,7 @@ def _fermionic_dipole(*args):
         """
         constants, integrals = dipole_integrals(mol, core, active)(*args)
 
-        nd = [anp.array([0]), anp.array([0]), anp.array([0])]
+        nd = [np.array([0]), np.array([0]), np.array([0])]
```

Comment:
I think here you can also use 'qml.math.array'

Reply:
Can we build `nd` as an array and then use vectorization inside the loop?

Reply:
> I think here you can also use 'qml.math.array'
Used `qml.math.array`

> Can we build nd as an array and then use vectorization inside the loop?
Probably we can, but just using `nd = qml.math.array([0, 0, 0])` fails some tests. Do you have a clean solution?